### PR TITLE
update doc for dtype & scheme customization

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

**Review by QuanluZhang — APPROVED** [review]

File: docs/en_US/Compression/Quantizer.rst [hunk 1]
```diff
@@ -102,6 +102,43 @@ The quantizer will automatically detect Conv-BN patterns and simulate batch norm
 graph. Note that when the quantization aware training process is finished, the folded weight/bias would be restored after calling
 `quantizer.export_model`.
 
+Quantization dtype and scheme customization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+QAT quantizer support dtype (int or uint) and scheme (per-tensor or per-channel and symmetric or affine) customization.
```
Comment (linbinskn) [inline docs/en_US/Compression/Quantizer.rst:107]: Suggest adding why we add dtype and scheme here. For instance, different inference frameworks support various and specific quantization strategies, nni aims to support different inference frameworks targeting popular devices.

Comment (chenbohua3) [reply docs/en_US/Compression/Quantizer.rst]: done

File: docs/en_US/Compression/Tutorial.rst [hunk 2]
```diff
@@ -83,30 +83,83 @@ when the value is int type, all quantization types share same bits length. eg.
...
+         'output': 'per_tensor_affine',
+         },
+   }
+
+when the value is str type, all quantization types share the same quant_scheme. eg.
+
+.. code-block:: bash
+
+   {
+      quant_scheme: 'per_channel_symmetric', # the quant_scheme of weight and output quantization are all 'per_channel_symmetric'
```
Comment (linbinskn) [inline docs/en_US/Compression/Tutorial.rst:126]: Should we also introduce four quant_scheme and two dtype briefly?

Comment (chenbohua3) [reply docs/en_US/Compression/Tutorial.rst:139]: done

**Review by linbinskn — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Pull request merged**